spec/datetimeformat.html

```diff
@@ -303,6 +320,12 @@ <h1>PartitionDateTimePattern ( _dateTimeFormat_, _x_ )</h1>
             1. Else,
               1. Let _fv_ be an implementation and locale dependent String value representing `"ante meridiem"`.
             1. Add new part record { [[Type]]: `"dayPeriod"`, [[Value]]: _fv_ } as a new element of the list _result_.
+          1. Else if _p_ is equal to `"fractionalSecondDigits"`, then
```

Comment:
This step needs to happen before step 20. e, otherwise it's not reachable. (Step 20.e is "If p matches a Property column of the row in Table 5, then", which also applies to `"fractionalSecondDigits"`.)